autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -426,7 +434,7 @@ def llm_call(
         raise ValueError("Failed to get a response from the LLM.")
 
     def _summarize_chunk(self, chunk: str, input_data: Input) -> str:
-        prompt = f"Summarize the following text concisely:\n\n{chunk}"
+        prompt = f"Summarize the following text in a {input_data.style} form. Focus your summary on the topic of `{input_data.focus}` if present, otherwise just provide a general summary:\n\n```{chunk}```"
```

Comment:
Please run `poetry run format`. This line and long `prompt = ` below may still require breaking into multiple lines manually, so lint passes.

Reply:
With our new docker setup is there a new way to do this, or do I have to poetry install just to format?